lib/presentation/di/module/network_module.dart

```diff
@@ -46,7 +47,7 @@ class NetworkModule {
   void _provideBaseOption() {
     final headers = <String, dynamic>{
       HttpHeaders.acceptHeader: 'application/json',
-      HttpHeaders.contentTypeHeader: 'application/json'
+      'content-type': 'application/json'
```

Comment:
This change has no effective, because `HttpHeaders.contentTypeHeader` are same 'content-type'

Reply:
no need to do that here. IMO